src/Config/Method.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace GravApi\Config;
+
+/**
+ * Class Method
+ * @package GravApi\Config
+ */
+class Method
+{
+    /**
+     * @var boolean
+     */
+    protected $enabled = false;
+
+    /**
+     * @var boolean
+     */
+    protected $useAuth = false;
```

Comment:
This should be changed to default to `true`. In the event that a user has _not specified_ whether they want to authenticate an endpoint/method, the API **should enforce it** to ensure data protection. In order to make something "public" it should be explicitly defined by the user.